group.cpp

```diff
@@ -16,22 +16,30 @@ namespace led
 /** @brief Overloaded Property Setter function */
 bool Group::asserted(bool value)
 {
+    if (customCallBack != nullptr)
+    {
+        // If lamp test is running (Asserted=true) and if user requests to stop
+        // lamp test (Asserted input=false), Stop operation will not take place
+        // and Set the Asserted to true itself. LampTest Asserted is/can be Set
+        // to false only when the lamptest timer expires.
```

Comment:
move this comment to the LampTest::requestHandler function

Reply:
done